go/encoding/basex/encoding.go

```diff
@@ -14,13 +14,14 @@ import (
 // An Encoding is a radix X encoding/decoding scheme, defined by X-length
 // character alphabet.
 type Encoding struct {
-	encode      []byte
-	decodeMap   [256]byte
-	inBlockLen  int
-	outBlockLen int
-	base        int
-	logOfBase   float64
-	baseBig     *big.Int
+	encode                 []byte
+	decodeMap              [256]byte
+	inBlockLen             int
+	outBlockLen            int
```

Comment:
`in` and `out` are a little ambiguous. Could we use names that are independent of the direction we're going? In the Python version I called them something like `bytesLen` and `charsLen`, though maybe there's something even better than "bytes" to disambiguate the base256 input?